specification/flag-evaluation.md

```diff
@@ -124,6 +111,17 @@ MyStruct myStruct = client.getObjectValue<MyStruct>('structured-flag', { text: '
 
 See [evaluation context](./evaluation-context.md) for details.
 
+###### Conditional Requirement 1.3.2.1
+
+> If languages differentiate between floats/ints, they **SHOULD** provide functions for both, consistent with language idioms.
+
+```go
+// example in GO
+GetIntValue(flag string, defaultValue int64, evalCtx EvaluationContext, options ...EvaluationOption) (int64, error)
+
+GetFloatValue(flag string, defaultValue float64, evalCtx EvaluationContext, options ...EvaluationOption) (float64, error)
+```
+
```

Comment:
`Conditional requirements` should be children of `Conditions`, so I think you want to put your "if" clause in a condition statement, and separate it from the requirement.

Reply:
I have fixed that.

I have also moved the `Conditional Requirement 1.3.3` to a **requirement** since it does not have any condition associated.